scripts/run_validator.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-tag=latest
+tag=change-view-submitted-response
```

Comment:
It was agreed to change view_submitted_response to view_response, due to the parent container already being called submission